### ISSUE TITLE
O&O ShutUp10++ bug in tweak section not working

### ISSUE THREAD

**Issue by ygoralmeida** [ChrisTitusTech/winutil#1955]
![Captura de tela 2024-05-09 163339](https://github.com/ChrisTitusTech/winutil/assets/6265324/d2901a57-70a5-4b77-a75c-b51f1ba10510)
![Captura de tela 2024-05-09 163355](https://github.com/ChrisTitusTech/winutil/assets/6265324/560ec302-10aa-4dc1-8f9c-70d89be7dbdc)
**Fully updated windows 11 23H2 - first time running the winutil script**
The bug appears while running the tweak section

Go to Tweaks and select Laptop, - make sure O&O is selected ( i've changed little bit the selection to enable or disable features, and never touched the O&O customization - just select the RUN Tweaks.

I got an alert error - that O&O started with wrong parameters or settings - in 09/05/2024 - 16:41 GMT -3 ( Brazil ) 

If not wrong, the error started like 3 weeks ago, since the last time i run the winutil script. Could not manage to fix that

> ====Chris Titus Tech=====
=====Windows Toolbox=====
Checking if Chocolatey is Installed...
===========================================
---      Chocolatey is installed        ---
===========================================
Version: v2.2.2
Running Script for WPFTweaksStorage
Set HKCU:\System\GameConfigStore\GameDVR_FSEBehavior to 2
Set HKCU:\System\GameConfigStore\GameDVR_Enabled to 0
Set HKCU:\System\GameConfigStore\GameDVR_DXGIHonorFSEWindowsCompatible to 1
Set HKCU:\System\GameConfigStore\GameDVR_HonorUserFSEBehaviorMode to 1
Set HKCU:\System\GameConfigStore\GameDVR_EFSEFeatureFlags to 0
HKLM:\SOFTWARE\Policies\Microsoft\Windows\GameDVR was not found, Creating...
Set HKLM:\SOFTWARE\Policies\Microsoft\Windows\GameDVR\AllowGameDVR to 0
Running Script for WPFTweaksRestorePoint
Changing system to allow multiple restore points per day
AVISO: O módulo 'Microsoft.PowerShell.Management' não foi importado porque o snap-in 'Microsoft.PowerShell.Management' já foi
importado.
System Restore Point Created Successfully
Running Script for WPFTweaksRightClickMenu
Restarting explorer.exe ...
Set HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\CapabilityAccessManager\ConsentStore\location\Value to Deny
Set HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Sensor\Overrides\{BFA794E4-F964-4FDB-90F6-51056BFE4B44}\SensorPermissionState to 0
Set HKLM:\SYSTEM\CurrentControlSet\Services\lfsvc\Service\Configuration\Status to 0
Set HKLM:\SYSTEM\Maps\AutoUpdateEnabled to 0
Set HKLM:\SYSTEM\CurrentControlSet\Services\Tcpip6\Parameters\DisabledComponents to 0
Running Script for WPFTweaksEnableipsix
Running Script for WPFTweaksOO
Applying recommended OO Shutup 10 Policies
Set HKLM:\SYSTEM\CurrentControlSet\Services\Tcpip6\Parameters\DisabledComponents to 1
Running Script for WPFTweaksTeredo
Setting Service AJRouter to Disabled
Setting Service ALG to Manual
Setting Service AppIDSvc to Manual
Setting Service AppMgmt to Manual
Setting Service AppReadiness to Manual
Setting Service AppVClient to Disabled
Setting Service AppXSvc to Manual
Setting Service Appinfo to Manual
Setting Service AssignedAccessManagerSvc to Disabled
Setting Service AudioEndpointBuilder to Automatic
Setting Service AudioSrv to Automatic
Setting Service Audiosrv to Automatic
Setting Service AxInstSV to Manual
Setting Service BDESVC to Manual
Setting Service BFE to Automatic
Setting Service BTAGService to Manual
Setting Service BcastDVRUserService_* to Manual
Setting Service BluetoothUserService_* to Manual
Setting Service BrokerInfrastructure to Automatic
Setting Service Browser to Manual
Setting Service BthHFSrv to Automatic
Setting Service CDPSvc to Manual
Setting Service CDPUserSvc_* to Automatic
Setting Service COMSysApp to Manual
Setting Service CaptureService_* to Manual
Setting Service CertPropSvc to Manual
Setting Service ClipSVC to Manual
Setting Service ConsentUxUserSvc_* to Manual
Setting Service CoreMessagingRegistrar to Automatic
Setting Service CredentialEnrollmentManagerUserSvc_* to Manual
Setting Service CryptSvc to Automatic
Setting Service CscService to Manual
Setting Service DPS to Automatic
Setting Service DcomLaunch to Automatic
Setting Service DcpSvc to Manual
Setting Service DevQueryBroker to Manual
Setting Service DeviceAssociationBrokerSvc_* to Manual
Setting Service DeviceAssociationService to Manual
Setting Service DeviceInstall to Manual
Setting Service DevicePickerUserSvc_* to Manual
Setting Service DevicesFlowUserSvc_* to Manual
Setting Service Dhcp to Automatic
Setting Service DiagTrack to Disabled
Setting Service DialogBlockingService to Disabled
Setting Service DispBrokerDesktopSvc to Automatic
Setting Service DisplayEnhancementService to Manual
Setting Service DmEnrollmentSvc to Manual
Setting Service Dnscache to Automatic
Setting Service DoSvc to AutomaticDelayedStart
Setting Service DsSvc to Manual
Setting Service DsmSvc to Manual
Setting Service DusmSvc to Automatic
Setting Service EFS to Manual
Setting Service EapHost to Manual
Setting Service EntAppSvc to Manual
Setting Service EventLog to Automatic
Setting Service EventSystem to Automatic
Setting Service FDResPub to Manual
Setting Service Fax to Manual
Setting Service FontCache to Automatic
Setting Service FrameServer to Manual
Setting Service FrameServerMonitor to Manual
Setting Service GraphicsPerfSvc to Manual
Setting Service HomeGroupListener to Manual
Setting Service HomeGroupProvider to Manual
Setting Service HvHost to Manual
Setting Service IEEtwCollectorService to Manual
Setting Service IKEEXT to Manual
Setting Service InstallService to Manual
Setting Service InventorySvc to Manual
Setting Service IpxlatCfgSvc to Manual
Setting Service KtmRm to Manual
Setting Service LSM to Automatic
Setting Service LanmanServer to Automatic
Setting Service LanmanWorkstation to Automatic
Setting Service LicenseManager to Manual
Setting Service LxpSvc to Manual
Setting Service MSDTC to Manual
Setting Service MSiSCSI to Manual
Setting Service MapsBroker to AutomaticDelayedStart
Setting Service McpManagementService to Manual
Setting Service MessagingService_* to Manual
Setting Service MicrosoftEdgeElevationService to Manual
Setting Service MixedRealityOpenXRSvc to Manual
Setting Service MpsSvc to Automatic
Setting Service MsKeyboardFilter to Manual
Setting Service NPSMSvc_* to Manual
Setting Service NaturalAuthentication to Manual
Setting Service NcaSvc to Manual
Setting Service NcbService to Manual
Setting Service NcdAutoSetup to Manual
Setting Service NetSetupSvc to Manual
Setting Service NetTcpPortSharing to Disabled
Setting Service Netman to Manual
Setting Service NgcCtnrSvc to Manual
Setting Service NgcSvc to Manual
Setting Service NlaSvc to Manual
Setting Service OneSyncSvc_* to Automatic
Setting Service P9RdrService_* to Manual
Setting Service PNRPAutoReg to Manual
Setting Service PNRPsvc to Manual
Setting Service PcaSvc to Manual
Setting Service PeerDistSvc to Manual
Setting Service PenService_* to Manual
Setting Service PerfHost to Manual
Setting Service PhoneSvc to Manual
Setting Service PimIndexMaintenanceSvc_* to Manual
Setting Service PlugPlay to Manual
Setting Service PolicyAgent to Manual
Setting Service Power to Automatic
Setting Service PrintNotify to Manual
Setting Service PrintWorkflowUserSvc_* to Manual
Setting Service ProfSvc to Automatic
Setting Service PushToInstall to Manual
Setting Service QWAVE to Manual
Setting Service RasAuto to Manual
Setting Service RasMan to Manual
Setting Service RemoteAccess to Disabled
Setting Service RemoteRegistry to Disabled
Setting Service RetailDemo to Manual
Setting Service RmSvc to Manual
Setting Service RpcEptMapper to Automatic
Setting Service RpcLocator to Manual
Setting Service RpcSs to Automatic
Setting Service SCPolicySvc to Manual
Setting Service SCardSvr to Manual
Setting Service SDRSVC to Manual
Setting Service SEMgrSvc to Manual
Setting Service SENS to Automatic
Setting Service SNMPTRAP to Manual
Setting Service SNMPTrap to Manual
Setting Service SSDPSRV to Manual
Setting Service SamSs to Automatic
Setting Service ScDeviceEnum to Manual
Setting Service Schedule to Automatic
Setting Service SecurityHealthService to Manual
Setting Service Sense to Manual
Setting Service SensorDataService to Manual
Setting Service SensorService to Manual
Setting Service SensrSvc to Manual
Setting Service SessionEnv to Manual
Setting Service SharedAccess to Manual
Setting Service SharedRealitySvc to Manual
Setting Service ShellHWDetection to Automatic
Setting Service SmsRouter to Manual
Setting Service Spooler to Automatic
Setting Service SstpSvc to Manual
Setting Service StateRepository to Manual
Setting Service StiSvc to Manual
Setting Service StorSvc to Manual
Setting Service SysMain to Automatic
Setting Service SystemEventsBroker to Automatic
Setting Service TabletInputService to Manual
Setting Service TapiSrv to Manual
Setting Service TextInputManagementService to Manual
Setting Service Themes to Automatic
Setting Service TieringEngineService to Manual
Setting Service TimeBroker to Manual
Setting Service TimeBrokerSvc to Manual
Setting Service TokenBroker to Manual
Setting Service TrkWks to Automatic
Setting Service TroubleshootingSvc to Manual
Setting Service TrustedInstaller to Manual
Setting Service UI0Detect to Manual
Setting Service UdkUserSvc_* to Manual
Setting Service UevAgentService to Disabled
Setting Service UmRdpService to Manual
Setting Service UnistoreSvc_* to Manual
Setting Service UserDataSvc_* to Manual
Setting Service UserManager to Automatic
Setting Service UsoSvc to Manual
Setting Service VGAuthService to Automatic
Setting Service VMTools to Automatic
Setting Service VSS to Manual
Setting Service VacSvc to Manual
Setting Service W32Time to Manual
Setting Service WEPHOSTSVC to Manual
Setting Service WFDSConMgrSvc to Manual
Setting Service WMPNetworkSvc to Manual
Setting Service WManSvc to Manual
Setting Service WPDBusEnum to Manual
Setting Service WSService to Manual
Setting Service WSearch to AutomaticDelayedStart
Setting Service WaaSMedicSvc to Manual
Setting Service WalletService to Manual
Setting Service WarpJITSvc to Manual
Setting Service WbioSrvc to Manual
Setting Service Wcmsvc to Automatic
Setting Service WcsPlugInService to Manual
Setting Service WdNisSvc to Manual
Setting Service WdiServiceHost to Manual
Setting Service WdiSystemHost to Manual
Setting Service WebClient to Manual
Setting Service Wecsvc to Manual
Setting Service WerSvc to Manual
Setting Service WiaRpc to Manual
Setting Service WinDefend to Automatic
Setting Service WinHttpAutoProxySvc to Manual
Setting Service WinRM to Manual
Setting Service Winmgmt to Automatic
Setting Service WlanSvc to Automatic
Setting Service WpcMonSvc to Manual
Setting Service WpnService to Manual
Setting Service WpnUserService_* to Automatic
Setting Service WwanSvc to Manual
Setting Service XblAuthManager to Manual
Setting Service XblGameSave to Manual
Setting Service XboxGipSvc to Manual
Setting Service XboxNetApiSvc to Manual
Setting Service autotimesvc to Manual
Setting Service bthserv to Manual
Setting Service camsvc to Manual
Setting Service cbdhsvc_* to Manual
Setting Service cloudidsvc to Manual
Setting Service dcsvc to Manual
Setting Service defragsvc to Manual
Setting Service diagnosticshub.standardcollector.service to Manual
Setting Service diagsvc to Manual
Setting Service dmwappushservice to Manual
Setting Service dot3svc to Manual
Setting Service edgeupdate to Manual
Setting Service edgeupdatem to Manual
Setting Service embeddedmode to Manual
Setting Service fdPHost to Manual
Setting Service fhsvc to Manual
Setting Service gpsvc to Automatic
Setting Service hidserv to Manual
Setting Service icssvc to Manual
Setting Service iphlpsvc to Automatic
Setting Service lfsvc to Manual
Setting Service lltdsvc to Manual
Setting Service lmhosts to Manual
Setting Service mpssvc to Automatic
Setting Service msiserver to Manual
Setting Service netprofm to Manual
Setting Service nsi to Automatic
Setting Service p2pimsvc to Manual
Setting Service p2psvc to Manual
Setting Service perceptionsimulation to Manual
Setting Service pla to Manual
Setting Service seclogon to Manual
Setting Service shpamsvc to Disabled
Setting Service smphost to Manual
Setting Service spectrum to Manual
Setting Service sppsvc to AutomaticDelayedStart
Setting Service ssh-agent to Disabled
Setting Service svsvc to Manual
Setting Service swprv to Manual
Setting Service tiledatamodelsvc to Automatic
Setting Service tzautoupdate to Disabled
Setting Service uhssvc to Disabled
Setting Service upnphost to Manual
Setting Service vds to Manual
Setting Service vm3dservice to Manual
Setting Service vmicguestinterface to Manual
Setting Service vmicheartbeat to Manual
Setting Service vmickvpexchange to Manual
Setting Service vmicrdv to Manual
Setting Service vmicshutdown to Manual
Setting Service vmictimesync to Manual
Setting Service vmicvmsession to Manual
Setting Service vmicvss to Manual
Setting Service vmvss to Manual
Setting Service wbengine to Manual
Setting Service wcncsvc to Manual
Setting Service webthreatdefsvc to Manual
Setting Service webthreatdefusersvc_* to Automatic
Setting Service wercplsupport to Manual
Setting Service wisvc to Manual
Setting Service wlidsvc to Manual
Setting Service wlpasvc to Manual
Setting Service wmiApSrv to Manual
Setting Service workfolderssvc to Manual
Setting Service wscsvc to AutomaticDelayedStart
Setting Service wuauserv to Manual
Setting Service wudfsvc to Manual
=================================
--     Tweaks are Finished    ---
=================================

>

**Comment by Marterich** [comment]
TLDR: Please check again after the next Release if the error still occurs

The OOSU logic tries to download the OOSU10.exe first and then afterward the config file containing the settings that OOSU should apply. If (for some reason) the download fails, OOSU10.exe tries to reference a config file that doesn't exist on startup. 

Seeing that I already created a PR that modifies the logic so that the config gets compiled right into winutil.ps1 so that no extra download (besides OOSU10.exe) is necessary, I'm fairly certain that nothing needs to be done here besides wait and try again. 
#1932
